wp-content/plugins/core/src/Blocks/Blocks_Subscriber.php

```diff
@@ -41,6 +41,10 @@ public function register(): void {
 				$style_override->register();
 			}
 		}, 10, 0 );
+
+		add_filter( 'block_categories', function ( array $categories ): array {
+			return $this->container->get( Block_Category::class )->custom_block_category( $categories );
+		}, 10, 2 );
```

Comment:
```suggestion
		}, 10, 1 );
```
This is only passing a single variable into the filter.